native/src/components/MapView.tsx

```diff
@@ -111,30 +92,27 @@ const MapView = ({
   const locationPermissionGrantedIcon = followUserLocation ? 'my-location' : 'location-searching'
   const locationPermissionIcon = locationPermissionGranted ? locationPermissionGrantedIcon : 'location-disabled'
 
-  const onPress = useCallback(
-    async (pressedLocation: Feature) => {
-      if (!mapRef.current || !cameraRef.current || !pressedLocation.properties) {
-        return
-      }
-      const featureCollection = await mapRef.current.queryRenderedFeaturesAtPoint(
-        [pressedLocation.properties.screenPointX, pressedLocation.properties.screenPointY],
-        undefined,
-        [featureLayerId]
-      )
-      const feature = featureCollection?.features.find((it): it is PoiFeature => it.geometry.type === 'Point')
-      if (feature) {
-        const {
-          geometry: { coordinates }
-        } = feature
-        setSelectedFeature(feature)
-
-        cameraRef.current.flyTo(coordinates)
-      } else {
-        setSelectedFeature(null)
-      }
-    },
-    [setSelectedFeature]
-  )
+  const onPress = async (pressedLocation: Feature) => {
+    if (!mapRef.current || !cameraRef.current || !pressedLocation.properties) {
+      return
+    }
+    const featureCollection = await mapRef.current.queryRenderedFeaturesAtPoint(
+      [pressedLocation.properties.screenPointX, pressedLocation.properties.screenPointY],
+      undefined,
+      [featureLayerId]
+    )
+    const feature = featureCollection?.features.find((it): it is PoiFeature => it.geometry.type === 'Point')
```

Comment:
In the Ehrenamtskarte App we select the feature which is closest to the location of the tap. Also: The first icon in the featureCollection is the one with the highest z index.

Maybe this could help if features are close together.

Reply:
i only found in type declaration for the symbolLayer
` symbolZOrder?: 'auto' | 'viewport-y' | 'source' | Expression;`
added ` symbolZOrder: 'source',`

Reply:
the function retrieving features
`mapRef.current.queryRenderedFeaturesAtPoint` has some kind of tolerance but when you click too far away the it results in an empty response
We can discuss if we really want to have this, because if we can not deselect any feature we never can show the POI List, so i don't think this is a good idea